doc/overview.md

```diff
@@ -0,0 +1,101 @@
+# GOV.UK Verify UK Proxy Node
+
+## Technical overview
+
+The UK Proxy Node allows GOV.UK Verify to provide identities as part of the eIDAS scheme. This document describes the architecture and behaviour of the UK Proxy Node.
+
+### Role of the UK Proxy Node
+As part of the eIDAS scheme, the UK must host a proxy node to allow users access to GOV.UK Verify when in an EU country. As defined in the eIDAS interoperability architecture, the proxy node will act as a front to one or more of a country’s identity schemes. 
+
+The UK Proxy Node will be introduced into the architecture, as a REST-based microservice, to conform with eIDAS specifications. 
+
+GOV.UK Verify uses a SAML message format that is different from the eIDAS specification. The UK Proxy Node supports the eIDAS SAML Profile and acts as an identity provider to EU member state services and as a Relying Party to GOV.UK Verify. 
+
+The role of the UK Proxy Node is to:
+ - convert eIDAS SAML to Verify SAML messages and vice versa
+ - validate all the incoming requests and outgoing responses 
+ - cryptographically sign all outgoing responses to EU services
+
```

Comment:
Can we add to Proxy Node role
 - encrypt the identity assertions sent to the EU service

Reply:
Sure. I'll add that.